cerberus-org/cerberus-app
# App sidenav

### Resolves #187 

Add an app-level sidenav that can be configured by dispatching an action.

![image](https://user-images.githubusercontent.com/2181467/34391309-608c1934-eb0a-11e7-8a17-362eaab61ea7.png)

## Testing

1. Verify that the sidenav only appears on the dashboard and is cleared properly on pages without sidenav options.
2. Verify that the page responds accordingly to screen size. The sidenav should be locked to the side on large screens and able to be opened/closed on xs.
